server/chat-plugins/lottery.js

```diff
@@ -0,0 +1,213 @@
+'use strict';
+
+const LOTTERY_FILE = 'config/chat-plugins/lottery.json';
+
+/** @type {typeof import('../../lib/fs').FS} */
+const FS = require(/** @type {any} */('../../.lib-dist/fs')).FS;
+
+/** @type {{[roomid: string]: {name: string, markup: string, participants: {[ip: string]: string}}}} */
+let lotteries = {};
```

Comment:
This should be `Object.create(null)` or `new Map()`, not `{}`. `{}` isn't actually an empty object

Reply:
But wouldn't that just not work once you load from the file and don't make a new object?